datafusion/core/src/physical_plan/mod.rs

```diff
@@ -402,6 +402,7 @@ pub mod union;
 pub mod unnest;
 pub mod values;
 pub mod windows;
+pub mod wrapper;
```

Comment:
```suggestion
mod wrapper;
```

I'm not sure this needs to be public